support/scripts/qemu-guest

```diff
@@ -778,8 +778,16 @@ case "$ARG_MACHINETYPE" in
 	"x86pc")
 		QEMU_BIN=${QEMU_BIN:-"$( which qemu-system-x86_64 )"}
 
+		# WORKAROUND: Unikraft currently only supports QEMUs PC model until 7.0
+		"${QEMU_BIN}" -M \? | grep -qe '^pc-i440fx-7\.0[[:space:]]\?'
```

Comment:
Can you reference the open issue here, since it is potentially fixable within the core?
```diff
  # WORKAROUND: Unikraft currently only supports QEMUs PC model until 7.0
+ # See: https://github.com/unikraft/unikraft/issues/1040
```

Reply:
Fine :+1: